src/MigrationTools/Configuration/EngineConfigurationBuilder.cs

```diff
@@ -85,30 +85,30 @@ public EngineConfiguration CreateEmptyConfig()
             {
                 LogLevel = LogEventLevel.Information,
                 Version = Assembly.GetExecutingAssembly().GetName().Version.ToString(2),
-                Source = new TeamProjectConfig()
-                {
-                    Project = "migrationSource1",
-                    AllowCrossProjectLinking = false,
-                    Collection = new Uri("https://dev.azure.com/nkdagility-preview/"),
-                    ReflectedWorkItemIDFieldName = "Custom.ReflectedWorkItemId",
-                    PersonalAccessToken = "",
-                    LanguageMaps = new LanguageMaps() { AreaPath = "Area", IterationPath = "Iteration" }
-                },
-                Target = new TeamProjectConfig()
-                {
-                    Project = "migrationTarget1",
-                    AllowCrossProjectLinking = false,
-                    Collection = new Uri("https://dev.azure.com/nkdagility-preview/"),
-                    ReflectedWorkItemIDFieldName = "Custom.ReflectedWorkItemId",
-                    PersonalAccessToken = "",
-                    LanguageMaps = new LanguageMaps() { AreaPath = "Area", IterationPath = "Iteration" }
-                },
                 FieldMaps = new List<IFieldMapConfig>(),
                 WorkItemTypeDefinition = new Dictionary<string, string> {
                     { "sourceWorkItemTypeName", "targetWorkItemTypeName" }
             },
                 Processors = new List<IProcessorConfig>()
             };
+            ec.Clients.Add(new TeamProjectConfig()
+            {
+                Project = "migrationSource1",
```

Comment:
Shouldnt there be a direction in these two configs?

Reply:
There is. I'm still working on this one so changed it to draft